src/app/feed/feed-card/feed-card.component.scss

```diff
@@ -108,8 +108,9 @@ a.post-link {
 	background-color: rgba(0, 0, 0, 0.5);
 
 	.overlay-content{
+		max-width: 90%;
+		height: 340px !important;
```

Comment:
It is processing the site. Please wait for a while. It takes almost 30 minutes to get site running.

https://brainbuzzer.github.io/loklak_search/